test/helpers/kubectl.go

```diff
@@ -1674,6 +1675,7 @@ func (kub *Kubectl) CiliumPreFlightCheck() error {
 // CiliumControllersPreFlightCheck validates that all controllers are not
 // failing. If any of the controllers fails will return an error.
 func (kub *Kubectl) CiliumControllersPreFlightCheck() error {
+	ginkgo.By("Checking that no controllers in Cilium have failed")
```

Comment:
I'm ok with all this By, but  be aware that maybe are too many and people may get confused 
I I would move this ones to the BeforeAll helper function and said something at the end.

`Cilium is ready, all preflight are ok and ready to test`

Reply:
@eloycoto the reason I added them here is that the PreFlightCheck does a lot of checks that have a large timeout. Adding it into each `BeforeAll` produces more boilerplate code that we have to copy and paste into each `BeforeAll`, which we can avoid by having such output put into the functions themselves. I think this is a happy medium between too little and too much output; it provides more visibility into the state of the checks without dumping a ton of output to console. However, we can remove these checks if they end up dumping a lot of non-useful output to console.